projects/igniteui-angular/src/lib/data-operations/data-util.ts

```diff
@@ -36,12 +40,18 @@ export class DataUtil {
             });
         return target;
     }
-    public static getFilteringConditionsForDataType(dataType: DataType):
-        {[name: string]: (value: any, searchVal?: any, ignoreCase?: boolean) => void} {
-        return FilteringCondition[dataType];
-    }
-    public static getListOfFilteringConditionsForDataType(dataType: DataType): string[] {
-        return Object.keys(DataUtil.getFilteringConditionsForDataType(dataType));
+    public static getFilteringConditionsForDataType(dataType: DataType): string[] {
```

Comment:
We should probably remove this from the API as it's not extendable anymore and not used anywhere either.